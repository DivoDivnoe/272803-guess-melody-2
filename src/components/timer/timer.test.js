import React from 'react';
import renderer from 'react-test-renderer';
import Timer from './timer.jsx';

describe(`Timer component`, () => {
  it(`is rendered correctly`, () => {
    const gameTime = 0;

    const tree = renderer.create(
        <Timer
          gameTime={gameTime}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

