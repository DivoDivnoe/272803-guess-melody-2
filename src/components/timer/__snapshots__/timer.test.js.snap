// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer component is rendered correctly 1`] = `
<div
  className="timer__value"
  style={
    Object {
      "color": "red",
    }
  }
  xmlns="http://www.w3.org/1999/xhtml"
>
  <span
    className="timer__mins"
  >
    00
  </span>
  <span
    className="timer__dots"
  >
    :
  </span>
  <span
    className="timer__secs"
  >
    00
  </span>
</div>
`;
